lib/upperkut/item.rb

```diff
@@ -4,47 +4,19 @@ module Upperkut
   class Item
     attr_reader :id, :body, :enqueued_at
 
-    def initialize(body:, id: nil, enqueued_at: nil)
-      raise ArgumentError, 'Body should be a Hash' unless body.is_a?(Hash)
-
+    def initialize(id:, body:, enqueued_at: nil)
```

Comment:
It is a strategy's responsibility to generate the item ID. Cloud Pubsub has a totally different approach to generate it than our Redis backed strategies.